### PR TITLE
#202 Add Back Button

### PR DESCRIPTION
**Pull request by ludwigschubi**
## Related Ticket/Issue
(DRIVE-202)[https://github.com/0wntech/drive/issues/202]

## Description
This PR adds:
* A 'Back'-Button that allows user to navigate to the previous folder

### CODE REVIEW

File: src/functional_components/BackButton/Backbutton.module.scss [hunk 1]
```diff
@@ -0,0 +1,23 @@
+@import '../../../styles/constants.scss';
+
```
Comment (bejow) [inline src/functional_components/BackButton/Backbutton.module.scss:2]: The back button above the folder content looks kind of strange. Should we add a drop shadow or something to round this up a bit? 
image

File: src/functional_components/BackButton/BackButton.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+export const BackButton = ({ setCurrentPath, currentPath, webId, history }) => {
+    const previousPath = getPreviousPath(currentPath);
+    return (
+        <div
+            className={classNames(styles.container, {
+                [styles.hidden]:
+                    currentPath && getRootFromWebId(webId) === currentPath,
+            })}
+            onClick={() => {
+                setCurrentPath(previousPath);
```
Comment (bejow) [inline src/functional_components/BackButton/BackButton.js:19]: Currently there is an error when clicking back in the root folder.
We should prevent that. Maybe don't display the back button if we're in the root directory.
image

File: src/utils/url.js [hunk 3]
```diff
@@ -33,6 +33,27 @@ export const getFileParamsFromUrl = (url) => {
...
+          })
+        : urlUtils.format({
+              ...urlObject,
+              pathname: pathname.replace(
+                  pathname.split('/')[pathname.split('/').length - 1],
+                  ''
+              ),
+          });
+};
+
```
Comment (bejow) [inline src/utils/url.js:56]: For good practice you could add a test for this function

File: src/functional_components/DropdownMenu/DropdownMenu.module.scss [hunk 4]
```diff
@@ -29,7 +28,7 @@
 
 .container {
     position: absolute;
-    top: 9px;
+    top: 5px;
```
Comment (bejow) [inline src/functional_components/DropdownMenu/DropdownMenu.module.scss:31]: top property could be removed completely, it prevents the container from centering

File: src/functional_components/BackButton/BackButton.js (hunk 2 above)
Comment (ludwigschubi) [reply src/functional_components/BackButton/BackButton.js:19]: should not be displayed in the root folder, will check this out

File: src/utils/url.js (hunk 3 above)
Comment (ludwigschubi) [reply src/utils/url.js:56]: true that

File: src/functional_components/DropdownMenu/DropdownMenu.module.scss (hunk 4 above)
Comment (ludwigschubi) [reply src/functional_components/DropdownMenu/DropdownMenu.module.scss:31]: yep

File: src/functional_components/BackButton/Backbutton.module.scss (hunk 1 above)
Comment (ludwigschubi) [reply src/functional_components/BackButton/Backbutton.module.scss:2]: Added a border, should be enough

File: src/utils/url.js (hunk 3 above)
Comment (ludwigschubi) [reply src/utils/url.js:56]: found getParentFolderUrl() will use that one instead, already has a test as well

**Comment by bejow** [comment]
@ludwigschubi can you resolve the merge conflicts?

**Pull request merged**